code/modules/overmap/missions/drill_mission.dm

```diff
@@ -0,0 +1,108 @@
+/datum/mission/drill
+	name = "Class 1 core sample mission"
+	desc = "We require geological information from one of the neighboring planetoids . \
+			Please anchor the drill in place and defend it until it has gathered enough samples.\
+			Operation of the core sampling drill is extremely dangerous, caution is advised. "
+	value = 2000
+	duration = 80 MINUTES
+	weight = 8
+
+	var/obj/machinery/drill/mission/sampler
+	var/num_wanted = 8
```

Comment:
Can you try lowering this to something like 6? Maybe 5?

Reply:
It has been done

Reply:
(I meant to do this back when I adjusted the amount of charges per vein but I forgot to)